src/client/src/commons/form/municipality/municipalityText.js

```diff
@@ -1,46 +1,19 @@
 import React from "react";
-import PropTypes from "prop-types";
 import _ from "lodash";
 
-import { getMunicipality } from "../../../api-lib/index";
-
 class MunicipalityText extends React.Component {
```

Comment:
Q: Siehe oben, braucht es das als separate Komponente?

Reply:
Im Grunde fehlt noch ein E2E Test für die ganze Geschichte.

Reply:
Den Tests habe ich in coordinates.cy.js integriert